android/app/src/main/java/org/fossasia/openevent/adapters/DayScheduleAdapter.java

```diff
@@ -107,6 +109,17 @@ public void onBindViewHolder(DayScheduleViewHolder holder, int position) {
         }
         holder.slotLocation.setText(currentSession.getMicrolocation().getName());
 
+        Track currentSessionTrack = currentSession.getTrack();
+        holder.slotTrack.setText(currentSessionTrack.getName());
```

Comment:
![10e42445-e1ee-4509-8ed7-e29668624cc4](https://cloud.githubusercontent.com/assets/19890557/26466537/ac441b54-41ac-11e7-90ee-b543919c874a.jpg)
@iamareebjamal  See this version too!